### DISCUSSION TITLE
milab v1

### DISCUSSION THREAD

**Discussion by arsenal3008** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#325] [Q&A]
Now the **imilab v1** robot vacuum cleaner has appeared, please support it.

**Comment by PiotrMachowski** [comment]
You have to create an issue and I will consider adding support for it

**Reply by arsenal3008** [reply]
I have such a robot vacuum cleaner and tried Xiaomi-Cloud-Map-Extractor it could not select maps

**Reply by PiotrMachowski** [reply]
You have to create an appropriate issue and provide all necessary data for me to even start doing anything related to this model.

**Reply by arsenal3008** [reply]
what data can i give?

**Reply by PiotrMachowski** [reply]
Everything is described in the issue creator

**Reply by arsenal3008** [reply]
I don't understand English well and I can't find the issue creator can you write what needs to be done?

**Reply by PiotrMachowski** [reply]
It is displayed when you click "Create" button in "Issues" tab. You have to do it for both Map Extractor and Map Card separately

**Reply by arsenal3008** [reply]
![2022-08-09_21-36-44](https://user-images.githubusercontent.com/6509294/183735930-afa5a52d-9ad5-44e4-9faf-03bda934eb1e.png)
this is?

**Reply by PiotrMachowski** [reply]
Yes
